ddtrace/contrib/grpc/utils.py

```diff
@@ -43,6 +44,12 @@ def set_grpc_method_meta(span, method, method_kind):
 def set_grpc_client_meta(span, host, port):
     if host:
         span.set_tag_str(constants.GRPC_HOST_KEY, host)
+        try:
+            _ = ipaddress.ip_address(host)
```

Comment:
Do we need to assign a variable at all here? 
```suggestion
            ipaddress.ip_address(host)
```